src_data/full_text.en.xml

```diff
@@ -140,6 +140,105 @@
 	<lakeMeadAnimated17>Consistent with the 2007 Interim Guidelines, a Level 3 Shortage Condition exists when Lake Mead's January 1 elevation is projected to be below an elevation of 1,025 feet. Annual delivery reductions for Nevada and Arizona are 20,000 acre-feet and 480,000 acre-feet, respectively. In addition, if Lake Mead is below an elevation of 1,025 feet, the Secretary of the Departement of Interior is required to consult with the seven Colorado River Basin States to consider further measures that may be taken under applicable Federal law. Yellow boxes indicate delivery reductions to states. Hover over the blue squares and states to see detailed delivery information.</lakeMeadAnimated17>
 	<lakeMeadAnimated18>Under the terms of Minute 319, when Lake Mead's January 1 elevation is projected to be below an elevation of 1,025 feet, Mexico's annual delivery may be reduced by 125,000 acre-feet. This reduction may be offset by delivery of water deferred under Minute 319 for a total annual delivery of 1.5 million acre-feet.</lakeMeadAnimated18>
 	<lakeMeadAnimated19>To learn more about Lower Colorado River water entitlement holders and consumptive use data from the Bureau of Reclamation, click button below.</lakeMeadAnimated19>
+	
+	<mead-pictogram-legend>Legend</mead-pictogram-legend>
+	
+	<mead-pictogram-legend-units>acre-feet</mead-pictogram-legend-units>
+		
+	<mead-scene-units>acre-feet/year</mead-scene-units>
+			
+	<Mexico-pictos-disclaimer>location of blue boxes does not represent location of water use.</Mexico-pictos-disclaimer>
+				
+	<x-pictogram-label>Lower Colorado River Water Entitlement Holders</x-pictogram-label>
+	
+	<x-pictogram-sub-label>(with an Average Annual Consumptive Use above 15,000 acre-feet)</x-pictogram-sub-label> 
```

Comment:
I included all punctuation that was within quotes from Jordan Read's file- up to him if he wants to leave it in or not. He was not consistent with including colons and parentheses.